inc/thorvg.h

```diff
@@ -1196,8 +1196,10 @@ class TVG_EXPORT SwCanvas final : public Canvas
      */
     enum Colorspace
     {
-        ABGR8888 = 0, ///< The channels are joined in the order: alpha, blue, green, red.
-        ARGB8888      ///< The channels are joined in the order: alpha, red, green, blue.
+        ABGR8888 = 0, ///< The channels are joined in the order: alpha, blue, green, red. Colors are alpha-premultiplied.
+        ARGB8888,     ///< The channels are joined in the order: alpha, red, green, blue. Colors are alpha-premultiplied.
+        abgr8888,     ///< The channels are joined in the order: alpha, blue, green, red. Colors are un-alpha-premultiplied. Alpha unmultiplication is an extra step, may be slower than premultiplied version.
```

Comment:
I think "Alpha unmultiplication is an extra step, may be slower than premultiplied version." this is not the proper description for users. They don't need to mind of it or, we should not support this option. So please remove them.

Reply:
I would like to suggest 2 options.
ABGR8888_STRAIGHT    @BETA_API
ARGB8888_STRAIGHT    @BETA_API

or  SwCanvas::premultiplied(on/off);      @BETA_API


What do you think?